akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -453,6 +480,17 @@ class ClusterSingletonManager(singletonProps: Props, terminationMessage: Any, se
   require(role.forall(cluster.selfRoles.contains),
           s"This cluster member [${cluster.selfAddress}] doesn't have the role [$role]")
 
+  private val singletonLeaseName = s"${context.system.name}-singleton-${settings.singletonName}"
```

Comment:
I was thinking about if you do a query for all leaves it would nice to be able to sort by actor system name, then purpose hence why i put system them and then singleton